javascript-challenges.test.js

```diff
@@ -13,21 +13,68 @@
 // --------------------1) Create a function that takes in a string and returns a string with a coded message. The coded message converts "a" to 4, "e" to 3, "i" to 1, and "o" to 0.
 
 // a) Create a test with expect statements using the variables provided.
+// describe('takes in a string and returns a string with a coded message', () => {
+//     it('should return a string with a coded message', () => {
+//       expect(secretCode(secretCodeWord1)).toEqual("L4ck4d41s1c4l");
+//       expect(secretCode(secretCodeWord2)).toEqual("G0bbl3dyg00k");
+//       expect(secretCode(secretCodeWord3)).toEqual("3cc3ntr1c");
+//     });
+//   });
+
+// // const secretCodeWord1 = "Lackadaisical"
+// // // Expected output: "L4ck4d41s1c4l"
+// // const secretCodeWord2 = "Gobbledygook"
+// // // Expected output: "G0bbl3dyg00k"
+// // const secretCodeWord3 = "Eccentric"
+// // // Expected output: "3cc3ntr1c"
+
+// // b) Create the function that makes the test pass.
+// // function secretCode = (string) => {
+// //     return input.replace(a, '4').replace(e, '3').replace(i, '1').replace(o, '0')
+// // }
+// function secretCode (input) {
+//     return input.replace(/a/g, '4').replace(/e/g, '3').replace(/i/g, '1').replace(/o/g, '0')
+// }
```

Comment:
Logic and pseudocode looks great!  Make sure to leave your code and tests commented in.